examples/api/test/painting/star_border/star_border.0_test.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'package:flutter_api_samples/painting/star_border/star_border.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  Finder getStartBorderFinder(StarBorder shape) {
+    return find.byWidgetPredicate(
+      (Widget widget) => widget is example.ExampleBorder && widget.border == shape,
+    );
+  }
+
+  testWidgets('Smoke test', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('Initial content is visible', (WidgetTester tester) async {
```
Or a better description